templates/pages/login.html.twig

```diff
@@ -34,97 +34,101 @@
-                  {{ languages_dropdown|raw }}
-               </div>
-            {% endif %}
+				{% if constant('GLPI_DEMO_MODE') %}
+					<div class="mb-3">
+						<label class="form-label" for="dropdown_language{{ rand }}">{{ __('Language') }}</label>
+						{{ languages_dropdown|raw }}
+					</div>
+				{% endif %}
 
-            {% if config('display_login_source') %}
-               <div class="mb-3">
-                  <label class="form-label" for="dropdown_auth{{ rand }}">{{ __('Login source') }}</label>
-                  {{ auth_dropdown_login|raw }}
-               </div>
-            {% endif %}
-
-            {% if config('login_remember_time') %}
-               <div class="mb-2">
-                  <label class="form-check" for="login_remember">
-                     <input type="checkbox" class="form-check-input" id="login_remember" name="{{ rmbfield }}" {{ config('login_remember_default') ? 'checked' : '' }} />
-                     <span class="form-check-label">{{ __('Remember me') }}</span>
-                  </label>
-               </div>
-            {% endif %}
-
-            <div class="form-footer">
-               <button type="submit" name="submit" class="btn btn-primary w-100" tabindex="3">
-                  {{ __('Sign in') }}
-               </button>
-            </div>
-         </div>
+				{% if config('display_login_source') %}
+					<div class="mb-3">
+						<label class="form-label" for="dropdown_auth{{ rand }}">{{ __('Login source') }}</label>
+						{{ auth_dropdown_login|raw }}
+					</div>
+				{% endif %}
 
-         {% if right_panel %}
-            <div class="col-auto px-2 text-center">
-               {% if text_login|length > 0 %}
-                  <div class="rich_text_container">
-                     {{ text_login|safe_html }}
-                  </div>
-               {% endif %}
+				{% if config('login_remember_time') %}
+					<div class="mb-2">
+						<label class="form-check" for="login_remember">
+							<input type="checkbox" class="form-check-input" id="login_remember" name="{{ rmbfield }}" {{ config('login_remember_default') ? 'checked' : '' }}/>
+							<span class="form-check-label">{{ __('Remember me') }}</span>
+						</label>
+					</div>
+				{% endif %}
 
-               {% if config('use_public_faq') %}
-                  <hr />
+				<div class="form-footer">
+					<button type="submit" name="submit" class="btn btn-primary w-100" tabindex="3">
+						{{ __('Sign in') }}
+					</button>
+				</div>
 
-                  <a class="btn btn-outline-secondary" href="front/helpdesk.faq.php">
-                     <i class="fas fa-question"></i>&nbsp;
-                     {{ __('FAQ') }}
-                  </a>
-               {% endif %}
+				{% if errors|length > 0 %}
+					<hr/>
+					<div class="alert alert-danger" role="alert">
+						{{ errors }}
+					</div>
+				{% endif %}
```

Comment:
`errors` variable is no longer defined here, as errors rendering has be moved to the `login_error.html.twig` template, see #16289.